modules/understanding-telemetry-and-insights-operator-data-flow.adoc

```diff
@@ -0,0 +1,17 @@
+// Module included in the following assemblies:
+//
+// * support/remote_health_monitoring/about-remote-health-monitoring.adoc
+
+[id="understanding-telemetry-and-insights-operator-data-flow_{context}"]
+= Understanding Telemetry and Insights Operator data flow
+
+The Insights Operator gathers data from the Kubernetes API and the Prometheus API into an archive. The archive is uploaded to link:https://cloud.redhat.com[cloud.redhat.com] every four minutes for processing. The Insights Operator also downloads the latest Insights analysis from link:https://cloud.redhat.com/openshift/[cloud.redhat.com]. This is used to populate the *Insights status* pop-up that is included in the *Overview* page in the {product-title} web console.
```

Comment:
Please switch the times for insights operator vs. telemetry.

Btw. the exact interval for telemetry is 4:30 minutes (https://github.com/openshift/telemeter/blob/0bb1b1ce693adcec3b5185c75641062168016eff/cmd/telemeter-client/main.go#L35), but probably doesn't matter for the sake of this document

Reply:
@iNecas thank you for your review!

Great catch! I  have now switched the timings and have been more specific about 4 minutes and 30 seconds.